layouts/error.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="layout-default">
+      <div class="pt-4 mt-12">
+        <center>
+            <div class="mt-12 pt-12">
+                <h1 v-if="error.statusCode === 404">This page could not be found</h1>
+                <h1 v-else>There was an unknown error</h1>
+            </div>
+        </center>
+      </div>
+      <div tyle="padding-bottom: 200px;" />
+      <br><br><br>
```

Comment:
@CryptoBender21 
Actually, I found that <br> had been used a lot but I think we should not use it.
We should use a specific value of margin as we do not want to repeat as many times as we need spacing.
Besides that, <br> might depend on browsers and OS so it's not reliable.

Reply:
I’m aware and I agree